src/server/rpc/procedures/eclipse-2017/eclipse-2017.js

```diff
@@ -0,0 +1,201 @@
+const Storage = require('../../../storage/storage'),
+    Logger = require('../../../logger'),
+    eclipsePathCenter = require('../../../../../utils/rpc/eclipse-2017/eclipsePathCenter.js'),
+    rpcUtils = require('../utils'),
+    stationUtils = require('./stations.js'),
+    logger = new Logger('netsblox:eclipse'),
+    storage = new Storage(logger);
+
+const STATIONS_COL = 'wuStations',
+    READINGS_COL = 'wuReadings';
+var connection,
+    latestReadings = {};
+
+let eclipsePath = function(){
+    return eclipsePathCenter();
+};
+
+// connect to nb database
+let dbConnect = () => {
+    if (!connection) {
+        connection = storage.connect();
```

Comment:
how could cause issues? it would cause an issue if only a bad query is submitted right? this same thing is going to apply if I use the rpc storage because I won't be using the existing helpers (get, save, remove).